files/beegfs-tuning.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=BeeGFS Tuning Service
+Documentation=http://www.beegfs.com/content/documentation/
+Requires=network-online.target
+After=network-online.target
+
+[Service]
+ExecStart=/bin/bash /opt/beegfs/beegfs-tuning.sh
```

Comment:
On a converged system, we might want to call this beegfs-oss-tuning - it's not about metadata or client, right?